packages/boxel/addon/components/boxel/input/ranged-number-picker/index.css

```
@@ -0,0 +1,45 @@
+.boxel-ranged-number-picker-menu {
+  display: inline-block;
+  background: url('/@cardstack/boxel/images/icons/caret-down.svg') no-repeat;
+  background-position: 100%;
+  padding-right: 1em;
+}
+
+.boxel-ranged-number-picker-menu
+  + .ember-basic-dropdown-content-wormhole-origin {
+  position: absolute;
+}
+
+.boxel-ranged-number-picker-menu__dropdown {
+  box-shadow: var(--boxel-box-shadow);
+  border-radius: var(--boxel-border-radius);
+  max-height: var(--boxel-xs-container);
+  overflow: auto;
+  margin-left: var(--boxel-sp-xs);
+}
+
+.boxel-ranged-number-picker-menu__dropdown ul {
+  list-style: none;
+  padding: 0;
+  text-align: center;
+}
+
+.ember-power-select-option[aria-current='true']
+  .boxel-ranged-number-picker-menu__item,
+.ember-power-select-option[aria-selected='true'] {
+  background-color: var(--boxel-light-100);
+}
+
+.boxel-ranged-number-picker-menu__item {
+  font-weight: 600;
+  font-size: var(--boxel-font-size-sm);
+  padding: var(--boxel-sp-xxs) var(--boxel-sp-sm);
+}
+
+.ember-power-select-placeholder,
```

```suggestion
.boxel-ranged-number-picker-menu .ember-power-select-placeholder,
```

I think this would prevent this style from applying to other uses of Ember Power Select outside of this component 🤓